src/test/scala/org/coroutines/regression-tests.scala

```diff
@@ -21,13 +21,11 @@ class RegressionTest extends FunSuite with Matchers {
     assert(c1.value == 5)
     assert(!c1.resume)
     assert(c1.isCompleted)
-    assert(c1.result == (()))
     val c2 = call(xOrY(-2, 7))
     assert(c2.resume)
     assert(c2.value == 7)
     assert(!c2.resume)
     assert(c2.isCompleted)
-    assert(c2.result == (()))
```

Comment:
My bad! Thought I got all of them before. Now I think I fixed all the ones [mentioned here](https://github.com/storm-enroute/coroutines/pull/22/commits/c6d06dcd241a89dfb86cbbeb4225bfb3bc679ea0#diff-f32e16f3115c779231d1f0b07d5182a9L85).